guides/installing-python.md

```diff
@@ -70,7 +69,6 @@ Per the [pyenv documentation](https://github.com/pyenv/pyenv#basic-github-checko
 export PYENV_ROOT="$HOME/.pyenv"
 
 # pyenv-virtualenvwrapper
-export PYENV_VIRTUALENVWRAPPER_PREFER_PYVENV="true"
```

Comment:
I removed this because it was causing problems for when when I added it. Instead of using `virtualenv`, it was prefering `pyvenv` in my default Python 3.6, and `pyvenv` is deprecated, and doesn't support passing in which Python to use for the virtualenv.